### ISSUE TITLE
Cannot reset voice packs to default in NVDA addon version 3.0 public beta 3

### ISSUE THREAD

**Issue by fastfinge** [tgeczy/NVSpeechPlayer#32]
Pressing reset to factory settings results in the error:
Factory defaults folder not found:
C:\Users\samue\AppData\Roaming\nvda\addons\tgSpeechBox\synthDrivers\tgSp...\.defaults
Cannot reset language packs.

**Comment by tgeczy** [comment]
Hey Sam — this is our fault, we didn't include the .defaults folder in the add-on package. The reset code looks for packs/.defaults/ and copies *.yaml back into packs/lang/, but that folder was missing from the build.

It's already documented in our release checklist, just got skipped in the beta 3 packaging. Next add-on build will have it. Thanks for catching it!
